feeds/wlan-ap/opensync/files/bin/lan-ssh-firewall

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+
+if [ "$#" -ne 1 ]; then
+    echo "Illegal number of parameters"
+    exit 1
+fi
+
+action=$1
+
+index=$(uci show firewall | grep "Lan SSH Rule" | cut -c 16-17)
```

Comment:
What if no rule?  Add the rule in ACCEPT state.

Reply:
I tested to see if the default firewall (uci default script) rule gets added in after a basic sysupgrade. It does appear to get run so we shouldn't have any issues where the firewall rule doesn't exist and we try and modify it.

Reply:
Cutting the 16 and 17th characters from the uci output is risky.  What if it is a 1 digit number?  And I am still worried that if the section is not present the script will fail or will modify some other section.